### PR TITLE
pythonPackages.dopy: refactor move to python-modules

### PR DESCRIPTION
**Pull request by costrouc**
###### Things done

pythonPackages.dopy: refactor move to python-modules

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/dopy/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  };
+
+  propagatedBuildInputs = [ requests six ];
+
+  meta = with pkgs.lib; {
+    description = "Digital Ocean API python wrapper";
+    homepage = "https://github.com/Wiredcraft/dopy";
+    license = licenses.mit;
+    maintainers = with maintainers; [ lihop ];
+    platforms = platforms.all;
```
Comment (FRidh) [inline pkgs/development/python-modules/dopy/default.nix]: Don't set platforms, it is set by buildPythonPackage

File: pkgs/development/python-modules/dopy/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ pkgs
```
Comment (FRidh) [inline pkgs/development/python-modules/dopy/default.nix]: use lib instead of pkgs

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/dopy/default.nix (hunk 2 above)
Comment (costrouc) [reply pkgs/development/python-modules/dopy/default.nix]: Glad to know this is the preferred attribute to pass. I will use that from now on. Thanks!

Comment (costrouc) [reply pkgs/development/python-modules/dopy/default.nix]: In this case `pkgs` is needed to use `fetchFromGithub`.

File: pkgs/development/python-modules/dopy/default.nix (hunk 1 above)
Comment (costrouc) [reply pkgs/development/python-modules/dopy/default.nix]: Fixed

**Comment by costrouc** [comment]
Fixed with corrections. Ready for merge.

**Comment by costrouc** [comment]
@GrahamcOfBorg build python2Packages.dopy python3Packages.dopy

**Comment by costrouc** [comment]
Ready for merge

**Pull request merged**